Misc/NEWS.d/next/Windows/Misc/NEWS.d/next/Windows/2018-09-07-10-57-00.bpo-34421.AKJISD.rst

```diff
@@ -0,0 +1,2 @@
+Fixed installation issue with setuptools setuptools that crashed with 
+packages containing unicode module or package names in Windows.
```

Comment:
The directories are messed up here.

Minor things:
- Can you put this in the Library section instead of Windows?  It is a distutils change on all platforms.
- Mention distutils, not setuptools.

Reply:
Done, thanks!